central/reportconfigurations/service/singleton.go

```diff
@@ -14,7 +15,8 @@ var (
 )
 
 func initialize() {
-	svc = New(reportConfigDS.Singleton(), notifierDataStore.Singleton(), accessScopeStore.Singleton(), manager.Singleton())
+	collectionDS, _ := collectionDataStore.Singleton()
```

Comment:
Odd that the singleton func should return an error ... and if it does, we shouldn't ignore it. panic(err).

Reply:
Ah, the singleton func returns both a datastore and a queryResolver instance. queryResolver has functions to generate a v1.Query from collection. Ignoring it here since config service just needs access to collection datastore.

Reply:
@keyallis Correct, the collectionDatastore's `Singleton` does error out on failure to build the collection graph. But it crashes on error and does not return any error.